api/src/main/resources/jira.yaml

```diff
@@ -0,0 +1,137 @@
+#This file provide the end points for jira interactions
+# JIRA REST API information can be found at :
+# https://developer.atlassian.com/cloud/jira/platform/rest/
+
+swagger: '2.0'
+
+info:
+  title: Jira
+  description: |
+    Service for Jira Interactions.
+  version: "0.1"
+
+host: "precisionmedicineinitiative.atlassian.net"
+schemes:
+  - "https"
+basePath: /rest/api/2
+
+produces:
+  - application/json
+
+securityDefinitions:
+  AdminSecurity:
+    type: basic
+
+##########################################################################################
+## PATHS
+##########################################################################################
+
+paths:
+ /issue:
+    post:
+      summary: Create issue in jira.
+      description: "Creates issue in JIRA and return IssueResponse"
+      tags:
+        - Jira
+      operationId: createIssue
+      parameters:
+        - name: fields
+          in: body
+          description: Attributes for creating issue
+          schema:
+            $ref: '#/definitions/IssueRequest'
+      responses:
+        200:
+          description: File saved
+          schema:
+            $ref: '#/definitions/IssueResponse'
+        400:
+          description: Bad request, check the summary/description field
+        503:
```

Comment:
I'm confused by the 503 responses here. Where does that come from? From the docs (https://developer.atlassian.com/cloud/jira/platform/rest/#api-api-2-issue-post), there's no mention of a 503 error. I think this should be a 500, along with the attachments call below.

Reply:
I encountered this  error while i was trying to create too many issues